# goal: Move the tool (e.g. camera) with given speed
#       parameters along given path
float32 maximum_velocity # in m/s
float32 acceleration # percentage of maximum
geometry_msgs/Pose tool_pose
geometry_msgs/PoseArray path 
---
# result
geometry_msgs/Pose end_pose
---
# feedback
geometry_msgs/Pose current_pose

